src/styled-components/Buttons/styles/index.js

```diff
@@ -0,0 +1,65 @@
+import { css } from "styled-components";
+
+const Base = css`
+  box-sizing: border-box;
+  cursor: pointer;
+  display: inline-block;
+  border: none;
```

Comment:
I prefere `border: 0`, but it's the same https://stackoverflow.com/questions/2922909/should-i-use-border-none-or-border-0